packages/core/src/embeddings/DeepInfraEmbedding.ts

```diff
@@ -0,0 +1,110 @@
+import { getEnv } from "@llamaindex/env";
+import { BaseEmbedding } from "./types.js";
+
+const DEFAULT_MODEL = "sentence-transformers/clip-ViT-B-32";
+
+const API_TOKEN_ENV_VARIABLE_NAME = "DEEPINFRA_API_TOKEN";
+
+const API_ROOT = "https://api.deepinfra.com/v1/inference";
+
+export interface DeepInfraEmbeddingResponse {
+  embeddings: number[][];
+  input_tokens: number;
```

Comment:
Currently there are 2 places where we return the number of input_tokens, it makes sense to leave only one. Pick based on what is done more commonly in other providers. Or put it in Status (it's metadata anyway).